galata/test/jupyterlab/sidebars.test.ts

```diff
@@ -30,6 +27,10 @@ async function mockLabelOnFirstTab(tabbar: Locator, text: string) {
     }, text);
 }
 
+test.use({
+  tmpPath: 'test-sidebars'
+});
```

Comment:
It looks like none of the sidebar tests passes anymore. Could it be related to the changes here (removal of `mockState: true` and addition of `tmpPath: 'test-sidebars'`)?

Reply:
Sidebar tests now pass, although other failures persist.